src/Devices/TempProbe_TC.cpp

```diff
@@ -89,3 +89,17 @@ void TempProbe_TC::setCorrection(float value) {
     serial(F("Set temperature correction to %i.%02i"), (int)correction, (int)(correction * 100 + 0.5) % 100);
   }
 }
+
+/**
+ * clearCorrection(f)
+ *
+ * Sets the correction offset back to 0
+ */
+void TempProbe_TC::clearCorrection() {
+  COUT("old = " << correction);
+  if (correction != 0) {
+    correction = 0.0;
+    EEPROM_TC::instance()->setCorrectedTemp(correction);
+    serial(F("Set temperature correction to %i.%02i"), (int)correction, (int)(correction * 100 + 0.5) % 100);
```

Comment:
You could just say that the new correction is "0.0", but this is fine as well.

Reply:
I like this because we would catch it if it’s bugged.